dashboard/app/controllers/api/v1/pd/workshop_organizer_survey_report_controller.rb

```diff
@@ -7,7 +7,7 @@ class WorkshopOrganizerSurveyReportController < ReportControllerBase
 
     # GET /api/v1/pd/workshop_organizer_survey_report_for_course/:course
     def index
-      workshops = ::Pd::Workshop.where(course: params[:course], organizer_id: current_user.id).in_state(::Pd::Workshop::STATE_ENDED)
+      workshops = ::Pd::Workshop.where(course: params[:course], organizer_id: current_user.id).in_state(::Pd::Workshop::STATE_ENDED).reject {|w| w.local_summer? || w.teachercon?}
```

Comment:
? Don't we only want to reject these when the `exclude_summer` param is set?

Can we do this in Sql? `reject` will first convert the `ActiveRecord::Relation` to an `Array` and inspect each model instance. What about adding something like a `excluding_summer` scope (with unit tests) that is added here when the param is present:

```ruby
workshops = workshops.excluding_summer if params[:exclude_summer]
```

Reply:
Moved to a scope, using the scope in both the survey controller and workshop API controller. I have to do it in both spots, because the survey results page calls for a list of workshops to populate the dropdown. I want to exclude summer workshops in that dropdown (you can get to those survey results from the main workshop dashboard)